extensions/amp-font/0.1/test/test-fontloader.js

```diff
@@ -22,19 +22,19 @@ import * as sinon from 'sinon';
 adopt(window);
 
 /** @private @const {string} */
-const FONT_FACE_ = '\
```

Comment:
yeah, didnt make the change here since i just used `lint --fix`. you want me to make the change?